src/components/policy/src/policy/include/policy/policy_manager_impl.h

```diff
@@ -297,7 +300,20 @@ class PolicyManagerImpl : public PolicyManager {
     TypeAccess CheckDriverConsent(const Subject& who, const Object& what,
                                   const std::string& rpc,
                                   const RemoteControlParams& params);
+    void CheckPTUUpdatesChange(
+      const utils::SharedPtr<policy_table::Table> pt_update,
+      const utils::SharedPtr<policy_table::Table> snapshot);
+    bool CheckPTURemoteCtrlChange(
+      const utils::SharedPtr<policy_table::Table> pt_update,
+      const utils::SharedPtr<policy_table::Table> snapshot);
 
+    void CheckPTUZonesChange(
+      const utils::SharedPtr<policy_table::Table> pt_update,
+      const utils::SharedPtr<policy_table::Table> snapshot);
+
+    void CheckRemoteGroupsChange(
```

Comment:
I suggest to use narrow types for these methods e.g.:
void CheckPTUZonesChange(
       const rpc::Optional<Equipment> old_equipment,
       const rpc::Optional<Equipment> new_equipment);